credentials.yaml.sample

```diff
@@ -1,4 +1,19 @@
 customer_credentials:
-  test-customer_id:
-      api_key: sample-api-key
-      model_name: gpt-3.5-turbo
+  testCustomerId:
+    credentialsMap:
+        AZURE_OPENAI:
+            customerId: testCustomerId
+            enabled: true
+            metadata:
+                deploymentName: gpt-4o
+                endpoint: https://myinstance.openai.azure.com/
+            secret: test_secret
+            type: AZURE_OPENAI
+        OPENAI:
+            customerId: testCustomerId
+            enabled: false
+            metadata:
+                model: gpt-3
+            secret: test_secret
+            type: OPENAI
+    customerId: testCustomerId
```

Comment:
stray line?

Reply:
wdym? it's an actual field in the yaml

Reply:
though we don't really use it so we could get rid of it for the sample